src/Compilers/Test/Resources/Core/CompilerTestResources.csproj

```diff
@@ -346,6 +346,7 @@
     <EmbeddedResource Include="SymbolsTests\snKey2.snk" />
     <EmbeddedResource Include="SymbolsTests\snPublicKey.snk" />
     <EmbeddedResource Include="SymbolsTests\snPublicKey2.snk" />
+    <EmbeddedResource Include="SymbolsTests\snMaxSizeKey.snk" />
```

Comment:
Is it possible to generate these keys vs. checking them in?  Trying to avoid a binary getting checked in here if possible.